numpy/lib/tests/test_index_tricks.py

```diff
@@ -262,6 +269,25 @@ def test_accepts_npfloating(self):
         assert_(grid32.dtype == np.float64)
         assert_array_almost_equal(grid64, grid32)
 
+    @pytest.mark.skipif(FLOAT128_EX, reason=str(FLOAT128_EX))
```

Comment:
If you use `np.longdouble` instead of `np.float128`, you don't need this marker.

Reply:
Good to know; implemented in https://github.com/numpy/numpy/pull/16971/commits/22789628d1cfe5e2ddb29a4466cbcd2224c85b67.